src/Defines.h

```diff
@@ -571,11 +571,7 @@ void AddFaceDirection(int & a_BlockX, int & a_BlockY, int & a_BlockZ, eBlockFace
 
 /** Returns the coords of a block that is neighboring the specified position through its specified face.
 If a_IsInverse is true, the opposite direction is used instead. */
-inline Vector3i AddFaceDirection(Vector3i a_Pos, eBlockFace a_BlockFace, bool a_bInverse = false)
-{
-	AddFaceDirection(a_Pos.x, a_Pos.y, a_Pos.z, a_BlockFace, a_bInverse);
-	return a_Pos;
-}
+Vector3i AddFaceDirection(Vector3i a_Pos, eBlockFace a_BlockFace, bool a_bInverse = false);
```

Comment:
having to write your function prototypes twice in an annoying thing

a_bInverse -> a_Inverse